io/plugins/eu.esdihumboldt.hale.io.xml/META-INF/MANIFEST.MF

```diff
@@ -13,8 +13,8 @@ Import-Package: com.google.common.collect;version="17.0.0",
  eu.esdihumboldt.util.resource,
  org.apache.http.client.fluent;version="4.3.6",
  org.slf4j;version="1.5.11"
-Bundle-ClassPath: lib/xercesImpl-2.9.1.jar,
- .
+Bundle-ClassPath: .
 Export-Package: eu.esdihumboldt.hale.io.xml.validator
 Require-Bundle: eu.esdihumboldt.hale.common.core;bundle-version="2.2.0",
- org.eclipse.core.contenttype;bundle-version="3.4.100"
+ org.eclipse.core.contenttype;bundle-version="3.4.100",
+ xerces.xercesImpl;bundle-version="2.9.1"
```

Comment:
Interesting that this still was there. I think the XercesValidator is not used anyway (instead the XMLApiValidator is used), so this should not have any effect on the XML Schema validation.